AprilSylph/XKit-Rewritten
# feat(Mutual Checker): Following icons in blog cards

### Description


This is really what I actually wanted the Mutual Checker blog card thing for: the ability to add an icon to blog cards when a user follows you but you don't follow them back as well as when they're a mutual. Oftentimes I'm in a thread somewhere and I see a blog that I think I might recognize, but that I'm not super familiar with as I don't follow them, and being able to see that they're a follower quickly is very convenient.

Obviously this doesn't work if the blog in question is a sideblog, but most of the time when I want this, it works great.

Why not do this for post headers as well? It would be an API request for every unique user whose post you see. Mutual Checker generally only fires API requests for each unique user **who you follow** whose posts you see, limiting impact.

### Testing steps


- Confirm that mutual icons are added to post headers and blog cards, that their colors are correct, and that they have tooltips ~~(except that the tooltips on post headers don't currently work, see #1779)~~.
- Confirm that following icons are added to blog cards of users that follow you, that their colors are correct, and that they have tooltips.
